Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,17 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+//  the header
+console.log("QTY     ITEM                TOTAL")
+// culculade total
+let grandTotalPence = 0;
+order.forEach(({itemName, quantity, unitPricePence })=>{
+  totalPrPenc = quantity * unitPricePence;
+  let grandTotalPence += totalPrPenc;
```

Comment:
In your recent change you added this `let` here. Why do you think you need it? On line 13 you already have variable with the same name. What will happen if you redeclare the same variable again? (adding `let` on line 16 is redeclaring the same variable)